teamA/jinwoo/routes/index.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const router = express.Router();
+
+const userRouter = require("./userRouter");
+router.use("/users", userRouter.router);
+
+const postRouter = require("./postRouter");
+router.use("/posts", postRouter.router);
+
+const likeRouter = require("./likeRouter");
+router.use("/like", likeRouter.router);
```

Comment:
```suggestion
const express = require("express");
const router = express.Router();

const userRouter = require("./userRouter");
const postRouter = require("./postRouter");
const likeRouter = require("./likeRouter");

router.use("/users", userRouter.router);
router.use("/posts", postRouter.router);
router.use("/like", likeRouter.router);
```

작성 잘해주셨습니다 :) 다음과 같이 수정하면 더 나은 가독성을 가진 코드가 될 것 같네요~~